lib/db.js

```diff
@@ -51,6 +52,7 @@ class LimitDBRedis extends EventEmitter {
     this.configurateBuckets(config.buckets);
     this.prefix = config.prefix;
     this.globalTTL = (config.globalTTL || ms('7d')) / 1000;
+    this.callCounts = new LRU({ max: 50 });
```

Comment:
Shouldn't this `max` be configurable?

Reply:
TBH; this will be used when a key has very low cardinality, I added an LRU just to not allow it to grow infinitely. But doesn't hurt to add a config.

Reply:
I can add a task to track this.

Reply:
```suggestion
    this.callCounts = new LRU({ max: config.maxCacheEntries || 50) });
```
Why are you setting 50? Don't you think this should be configurable?